### PR TITLE
ZUR-94: Fix the padding issue in the channel name

### PR DESCRIPTION
**Pull request by The-Legend-Developer**
There is too much space between the arrow and the channel name

### CODE REVIEW

**Comment by billmal071** [comment]
Add a picture to show what you did.

**Comment by Michaelajayi150** [comment]
The task was to fix the "padding" issue between the channel name as noted below
The gap in the channel name to look exactly as it is in the original slack app

How it is meant to be

The issue was not a padding issue but a **margin left** issue that was added to the class of the arrow icon
So I removed the **margin left** on the arrow icon...

**Comment by JulianaSau** [comment]
@The-Legend-Developer Where's the screenshot of how it looks after you've made the changes?

**Comment by Michaelajayi150** [comment]
After removing the **margin-left** property
After solving

**Review by JulianaSau — APPROVED** [review]

**Review by JulianaSau — COMMENTED** [review]
Could you also add a small space between the # and the channel name

**Comment by Michaelajayi150** [comment]
@JulianaSau Okay, Will do that now

**Comment by JulianaSau** [comment]
Okay. You'll inform me once you've made the changes

**Comment by The-Legend-Developer** [comment]
The spacing issue was coming from the props received by the component itself. The issue was that the **props.name** was a string that has **#** embedded in it. I didn't want to change the props received so I splice the string and added a space to the output string....
Changes are below:
Before changes

The code to add space in the channel name at the top
Code to change output string

The change after:
Issue fixed

I add some problems while pushing to Github upon a outdated upstream so I first had to revert my code then update the codebase back hence the reverted commits.

**Comment by The-Legend-Developer** [comment]
@JulianaSau All done

**Comment by JulianaSau** [comment]
@The-Legend-Developer, pull the latest changes from the repo, resolve your conflicts then tag me when you're done

File: .github/workflows/prod-deployment.yaml [hunk 1]
```diff
@@ -28,4 +28,4 @@ jobs:
             git pull origin main
             yarn install
             yarn build
-            pm2 restart zuri-main
+            pm2 restart zc_main_prod
```
Comment (JulianaSau) [inline .github/workflows/prod-deployment.yaml:31]: Why did you change this file?

**Review by JulianaSau — CHANGES_REQUESTED** [review]
There are so many changes on this pr that aren't related to your task. I only expect changes to at most two files for your task. Either close this pr and create one that has the right commits or fix this one to have the right commits.

File: .github/workflows/prod-deployment.yaml (hunk 1 above)
Comment (Michaelajayi150) [reply .github/workflows/prod-deployment.yaml:31]: I didn't... It was fetched when I pulled request from GitHub all those files change I thought It was from your end but when I figured that's when I reverted the changes

Comment (JulianaSau) [reply .github/workflows/prod-deployment.yaml:31]: Okay then. I suggest closing this or then opening another one with clean commits so that I can merge it. Your functionality is actually working so it's these stray commits that are delaying the merging @The-Legend-Developer
